src/github/client/github-client.ts

```diff
@@ -15,20 +28,37 @@ export class GitHubClient {
 	protected readonly axios: AxiosInstance;
 
 	constructor(
+		gitHubConfig: GitHubConfig,
 		logger: Logger = getLogger("gitHub-client"),
-		baseUrl?: string
+		baseUrl?: string // goes away when we remove FF
 	) {
 		this.logger = logger;
 
+		// constructor cannot be async, therefore cannot use await
+		booleanFlag(BooleanFlags.USE_GITHUB_CONFIG_IN_BASE_CLIENT, false)
```

Comment:
I'm not sure an FF is really needed for this change as it would be thoroughly tested in unit test and pollinator.  Any problem here would fail spectacularly.  There's also the issue of potential race conditions with this...

Reply:
According to https://hello.atlassian.net/wiki/spaces/BPP/pages/128871659/Feature+Delivery+FAQ , we should use feature flags unless there's a good excuse not to ¯\_(ツ)_/¯